index.md

```diff
@@ -17,14 +17,14 @@ prominent role in representing The Carpentries community online.
 You will develop skills for creating a positive environment for all contributors.
 
 - Communicating with other Maintainers: As a new Maintainer,
-you are part of a large team supporting over 40 official lessons. You will learn how to 
+you are part of a large team supporting 48 official lessons. You will learn how to 
```

Comment:
Same comment as before: nice to have the exact number, but requires updating.

Reply:
```suggestion
you are part of a large team supporting over 40 official lessons. You will learn how to 
```